syndesis/src/app/App.tsx

```diff
@@ -29,50 +40,76 @@ export interface IAppBaseProps {
 export class App extends React.Component<IAppBaseProps> {
   public render() {
     return (
-      <WithConfig>
-        {({ config, loading, error }) => (
-          <WithLoader
-            loading={loading}
-            error={error}
-            loaderChildren={<Loader />}
-            errorChildren={<UnrecoverableError />}
-            minWait={1000}
-          >
-            {() => (
-              <AppContext.Provider
-                value={{ config: config!, logout: this.logout }}
+      <NamespacesConsumer ns={['app']}>
+        {t => (
+          <WithConfig>
+            {({ config, loading, error }) => (
+              <WithLoader
+                loading={loading}
+                error={error}
+                loaderChildren={<Loader />}
+                errorChildren={<UnrecoverableError />}
+                minWait={1000}
               >
-                <ApiContext.Provider
-                  value={{
-                    apiUri: `${config!.apiBase}${config!.apiEndpoint}`,
-                    headers: { 'SYNDESIS-XSRF-TOKEN': 'awesome' },
-                  }}
-                >
-                  <ApiContext.Consumer>
-                    {({ apiUri, headers }) => (
-                      <WithServerEvents apiUri={apiUri} headers={headers}>
-                        {functions => (
-                          <ServerEventsContext.Provider value={functions}>
-                            <Layout navbar={this.renderNavbar()}>
-                              <React.Fragment>
-                                <Switch>{this.renderRoutes()}</Switch>
-                              </React.Fragment>
-                            </Layout>
-                          </ServerEventsContext.Provider>
+                {() => (
+                  <AppContext.Provider
+                    value={{ config: config!, logout: this.logout }}
+                  >
+                    <ApiContext.Provider
+                      value={{
+                        apiUri: `${config!.apiBase}${config!.apiEndpoint}`,
+                        headers: { 'SYNDESIS-XSRF-TOKEN': 'awesome' },
+                      }}
+                    >
+                      <ApiContext.Consumer>
+                        {({ apiUri, headers }) => (
+                          <WithServerEvents apiUri={apiUri} headers={headers}>
+                            {functions => (
+                              <ServerEventsContext.Provider value={functions}>
+                                <AppLayout
+                                  appTitle={'Syndesis'}
```

Comment:
@gashcrumb @elvisisking we handle the app name in a special way in the Angular app. What should we do here?

Reply:
In the angular app it's in the localization file, but then we override it by setting it from the `config.json`, which then lets us easily override the name for the productized version.  We should do something similar here I think, or at least pull it from the `config.json` for starters.

Might be worth doing a stopgap solution for now and we can create an issue and collect requirements maybe?

Reply:
heh, kicked the can down the road :-)